src/ipv4.rs

```diff
@@ -253,22 +253,31 @@ impl From<Ipv4Addr> for Ipv4Network {
     }
 }
 
+#[derive(Copy, Clone, Debug)]
```

Comment:
Should probably not implement `Copy`?

I say this because I have seen this comment: https://github.com/rust-lang/rust/issues/58520#issuecomment-553682166
However, I don't find the underlying decision or discussion about it. But it seems reasonable that they would be confusing. That since every time you pass it around you get more iterators that start off from the same place.

Reply:
Good point, I've removed it.